Bonsai.Tinkerforge/BrickletGPSV2.cs

```diff
@@ -0,0 +1,211 @@
+﻿using System;
+using System.ComponentModel;
+using System.Reactive.Disposables;
+using System.Reactive.Linq;
+using Tinkerforge;
+
+namespace Bonsai.Tinkerforge
+{
+    [DefaultProperty(nameof(Uid))]
+    [Description("Measures coordinates, altitude, modtion, timestamp, satellite status from a GPS Bricklet 2.0.")]
+    public class BrickletGPSV2 : Combinator<IPConnection, Tuple<BrickletGPSV2.StatusData, BrickletGPSV2.CoordinateData, BrickletGPSV2.AltitudeData, BrickletGPSV2.DateTimeData>>
+    {
+        [Description("The unique bricklet device UID.")]
+        [TypeConverter(typeof(UidConverter))]
+        public string Uid { get; set; }
+
+        [Description("Specifies the period between sample event callbacks. A value of zero disables event reporting.")]
+        public long Period { get; set; } = 1000;
+
+        [Description("If SBAS is enabled, the position accuracy increases (if SBAS satellites are in view), but the update rate is limited to 5Hz. With SBAS disabled the update rate is increased to 10Hz.")]
+        public SBASConfig SBAS { get; set; } = SBASConfig.Disabled;
```

Comment:
It might be helpful if we spell-out what the SBAS acronym stands for at the beginning of the documentation string. Alternatively, we can include it in the extended documentation later, in which case it might be fine to leave it like this for now.